coralproject/cay
# Drag and drop refactor

## What does this PR do?
- Adds drag and drop functionality to Form Creator

## How do I test this PR?
- Go to Form Creator
- Try adding fields to the form by dragging them
- Try re-sorting the iist by dragging the fields

Save or preview the form: field order should be the same as the one in the form builder.

@coralproject/frontend
